test/arbitrary.test.ts

```diff
@@ -14,6 +16,7 @@ describe('Arbitrary tests', () => {
 
   it('should return values in the specified range', () => {
     expect(fc.scenario()
+      .config(new Strategies.RandomCachedStrategy())
```

Comment:
~Pretty please for a default strategy xD~. Just realised _there is_ a default strategy, which is `BiasedRandomChachedStrategyWithShrinking`. I don't enjoy this code duplication. Can't you use something like `beforeEach` from mocha to initialize this recurrent scenario?

Reply:
Yeah, I was not liking it too. I will investigate this.

Reply:
According to @arestivo suggestions, I will create a test suite specifically for strategies and also change the way strategies are configured. Therefore, what is mentioned here is no longer a problem.